src/views/add-collaborator.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.html" %}
+
+{% block beforeContent %}
+  {% include "include/back-link.html" %}
+{% endblock %}
+
+{% block pageContent %}
+  <h1 class="govuk-heading-l">Add a New Collaborator to a Repository</h1>
+
+  <p class="govuk-body">
+    When you want to grant more granular access to a repository, you can add an external collaborator directly to the repository.
```

Comment:
I would re-phrase this sentence as it isn't clear that outside collaborators aren't part of the org and only have repository rights. Use [GitHub docs](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization) as a template